extensions/amp-form/0.1/amp-form.js

```diff
@@ -443,7 +377,7 @@ function getUserValidityStateFor(element) {
 function checkUserValidity(element, propagate = false) {
```

Comment:
I am just trying to make a distinction between the two because of the browser support gap in one and not the other. All browsers support validating the input (checking if the input is valid or not) but most a lot don't support reporting (telling you what is wrong with the input).

There isn't really two concepts here just was trying to make the distinction.